libs/langchain-google-common/src/types.ts

```diff
@@ -91,6 +91,15 @@ export interface GoogleAIModelParams {
   safetySettings?: GoogleAISafetySetting[];
 
   convertSystemMessageToHumanContent?: boolean;
+
+  /**
+   * Available for gemini-1.5-pro.
+   * The output format of the generated candidate text.
+   * Supported MIME types:
+   *     text/plain: (default) Text output.
+   *     application/json: JSON response in the candidates.
+   */
+  responseMimeType?: "text/plain" | "application/json"
```

Comment:
Good to see the comment!
I would suggest making the two strings a separate type and using that type here and down in `GeminiGenerationConfig`.

Reply:
definitely I will